statsmodels/discrete/tests/test_constrained.py

```diff
@@ -528,6 +553,38 @@ def setup_class(cls):
         cls.constraints_rq = (R, q)
 
 
+class TestLogitConstrained2HC(CheckGLMConstrainedMixin):
+
+    @classmethod
+    def setup_class(cls):
+        cls.idx = slice(None)  # params sequence same as Stata
+        #res1ul = Logit(data.endog, data.exog).fit(method="newton", disp=0)
+        cls.res2 = reslogit.results_constraint2_robust
+
+        mod1 = Logit(spector_data.endog, spector_data.exog)
+
+        # not used to match Stata for HC
+        # nobs, k_params = mod1.exog.shape
+        # k_params -= 1   # one constraint
+        cov_type = 'HC0'
+        cov_kwds = {'scaling_factor': 32/31}
+        # looks like nobs / (nobs - 1) and not (nobs - 1.) / (nobs - k_params)}
```

Comment:
any speculation on why?

Reply:
I don't remember specifically here, but Stata and I/statsmodels don't always agree on df correction, e.g. in robust cov_type for GLM